content/applications/finance/accounting/receivables/customer_invoices/electronic_invoicing.rst

```diff
@@ -0,0 +1,82 @@
+================================================================
+Electronic invoicing (:abbr:`EDI (electronic data interchange)`)
+================================================================
+
+EDI, or electronic data interchange, is the inter-company communication of business documents, such
+as purchase orders and invoices, in a standard format. Sending documents according to an EDI
+standard ensures that the machine receiving the message can interpret the information correctly.
+Various EDI file formats exist and are available depending on your company's country.
+
+EDI feature enables automating the administration between companies and might also be required by
+some governments for fiscal control or to facilitate the administration.
+
+Any business that processes large volumes of documents and forms can benefit from an EDI solution to
+increase automation and improve accuracy.
+
+Odoo supports, among others, the following formats.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Format Name
+     - Applicability
+   * - Factur-X (CII)
+     - Default format on Odoo (enabled by default)
+   * - Peppol BIS Billing 3.0 (UBL)
+     - For companies which countries are part of the `EAS list
+       <https://docs.peppol.eu/poacc/billing/3.0/codelist/eas/>`_
+   * - E-FFF
+     - For Belgian companies
+   * - XRechnung (UBL)
+     - For German companies
+   * - Fattura PA (IT)
+     - For Italian companies
+   * - CFDI (4.0)
+     - For Mexican companies
+   * -  Peru UBL 2.1
+     - For Peruvian companies
+   * - SII IVA Llevanza de libros registro (ES)
+     - For Spanish companies
+
+.. seealso::
+   - :doc:`../../fiscal_localizations/overview/fiscal_localization_packages`
+   - :doc:`../../fiscal_localizations/overview/localizations_list`
```

Comment:
If the list above isn't more detailed, I think this link may be misleading, since it may suggest we're talking about EDI supported according to the countries listed. It isn't a great page anyway and I wouldn't promote it.

Reply:
I removed the links then